Rocket.Chat.iOS/ViewControllers/RegisterViewController.swift

```diff
@@ -66,22 +105,105 @@ class RegisterViewController: UIViewController {
             emailTextField.layer.borderColor = UIColor.redColor().CGColor
             emailTextField.layer.borderWidth = 1
             
+            //Create View Controller
+            let popoverVC = storyboard?.instantiateViewControllerWithIdentifier("emailPopover")
+            
+            //Set it as popover
+            popoverVC!.modalPresentationStyle = .Popover
+            
+            //Set the size
+            popoverVC!.preferredContentSize = CGSizeMake(250, 50)
+            
+            
+            if let popoverController = popoverVC!.popoverPresentationController {
+                
+                //Specify the anchor location
+                popoverController.sourceView = emailTextField
+                popoverController.sourceRect = emailTextField.bounds
+                
+                
+                //Popover above the textfield
+                popoverController.permittedArrowDirections = .Down
+                
+                //Set the delegate
+                popoverController.delegate = self
+            }
+            
+            //Show the popover
+            presentViewController(popoverVC!, animated: true, completion: nil)
+
         }
         
         //then password check
-        else if (passwordTextField.text!.isEmpty || passwordTextField.text!.characters.count < 8) {
+        else if (passwordTextField.text!.characters.count < 8) {
```

Comment:
As I say in the commit, this is just for demo purposes. Checking the password probably will happen on the server side...right?